### PR TITLE
Deprecate gpsupport

### PR DESCRIPTION
**Pull request by janebeckman**
Also removes notice gpcheck is deprecated and use gpsupport instead.

### CODE REVIEW

File: gpdb-doc/dita/admin_guide/perf_troubleshoot.xml [hunk 1]
```diff
@@ -166,24 +166,17 @@ con6 /gpdata/*/pg_log/gpdb*.csv' &gt; seglog.out
...
-                        format="html" class="- topic/xref ">Pivotal Documentation</xref> site.</p>
+                <p>The Greenplum Magic Tool (GPMT) utility can run diagnostics and collect
+                    information from a Greenplum Database system. You can then send the information
+                    to Pivotal Customer Support to aid the diagnosis of Greenplum Database errors or
+                    system failures.</p>
+                <p>The GPMT  utility is available from the <xref
+                        href="https://discuss.pivotal.io/hc/en-us/categories/200072608-Pivotal-Greenplum-Knowledge-Base"
+                        format="html" scope="external">Pivotal Knowledge Base</xref> on the <xref
+                        href="https://discuss.pivotal.io/hc/en-us/articles/217546277-Greenplum-Magic-Tool-GPMT-"
+                        format="html" scope="external">GPMT</xref> page.</p>
```
Comment (danielgustafsson) [inline gpdb-doc/dita/admin_guide/perf_troubleshoot.xml]: This seems like something that should be in the Pivotal Greenplum Enterprise documentation and not in the open source docs?

**Comment by janebeckman** [comment]
Something you probably don't see in the pull request is that the source for
GPMT is actually tagged with conditional text for Pivotal-only, since we
single-source this in DITA XML. David is probably better qualified to
address the issue of separate documentation.

**Comment by dyozie** [comment]
Right - we use conditional text in the XML so that any pivotal-specific
mentions don't get built into the Open Source documentation.  The reverse
case is also used so that, for example, OSS-specific build instructions
don't appear in the proprietary doc builds.

-David

**Comment by cjcjameson** [comment]
Confirming: the current practice is that Pivotal-specific mentions will be at various places throughout the markup language for the documentation, using `otherprops="pivotal"` to designate the sections.

**Comment by dyozie** [comment]
That's correct.  The converse, otherprops="oss-only" would be used to
prevent an element from appearing in the Pivotal build.

File: gpdb-doc/dita/admin_guide/perf_troubleshoot.xml (hunk 1 above)
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/perf_troubleshoot.xml]: otherprops="pivotal" in the enclosing topic conditionalizes all of this for the proprietary build.  Conversely, otherprops="oss-only" can be used for stuff that doesn't apply to the proprietary case, like build instructions.

**Pull request merged**